src/lib/generateID.ts

```diff
@@ -1,18 +1,11 @@
-import getRandomValues from 'get-random-values';
+import crypto from 'crypto';
 
 /**
  * Creates an Interclip-compatible random ID
  * @param length the length of the ID to generate
  * @returns a random ID
  */
-export const getRandomID = (length: number): string => {
-  return [...getRandomValues(new Uint8Array(length))]
-    .map(
-      (x, i) => (
-        (i = ((x / 255) * 61) | 0),
-        String.fromCharCode(i + (i > 9 ? (i > 35 ? 61 : 55) : 48))
-      ),
-    )
-    .join('')
-    .toLowerCase();
+export const getClipHash = (url: string): string => {
+  const digest = crypto.createHash('sha512').update(url).digest('hex');
+  return BigInt('0x' + digest).toString(36);
```

Comment:
bigint ❤️

Reply:
That's exactly what popped into my head 😄

Reply:
Ethereum uses bigint 😄

Reply:
Welp Interclip now does too!

Reply:
Then let us build Interclip blockchain 🤣